docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,171 @@
+
+    ```csharp
+    string message = "      Hello World!       ";
+    Console.WriteLine("[" + message + "]");
+
+    string trimmedMessage = message.TrimStart();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.TrimEnd();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.Trim();
+    Console.WriteLine("[" + trimmedMessage + "]");
+    ```
+
+    This sample reinforces a couple important concepts for working with strings. The methods that manipulate strings return new string objects rather than making modifications in place. You can see that each call to any of the Trim methods returns a new string, but does not change the original message.
+
+    Other methods perform more general edits on a string. You've probably used a search and replace command in an editor or word processor. The Replace method substitutes a replacement string for all occurences of a sub-string in a string. Try it for yourself. Add this code above:
+
+    ```csharp
+    string message = "Hello World!";
+    Console.WriteLine(message);
+    message = message.Replace("Hello", "Greetings");
+    Console.WriteLine(message);
+    ```
+
+    Two other useful methods make a string ALL CAPS or all lower case. Try this code above:
+
+    ```csharp
+    string message = "Hello World!";
+    Console.WriteLine(message);
+    Console.WriteLine(message.ToUpper());
+    Console.WriteLine(message.ToLower());
+    ```
+- title: Search strings
+  durationInMinutes: 4
+  content: |
+    The other part of a *search and replace* operation is to find text in a
+    sring. You can use the the `Contains` method for searching. It
+    tells you if a method contains a sub-string inside it. Try the following
+    code to explore `Contains`:
+
+    ```csharp
+    string message = "You say goodbye, and I say hello";
+    Console.WriteLine(message.Contains("goodbye"));
+    Console.WriteLine(message.Contains("greetings"));
+    ```
+
+    The `Contains` method returns a **boolean** value which tells you if the
+    string you sought was found. A **boolean** stores either a `true` or a
+    `false` value. You'll learn more about **boolean** values
+    in a later lesson.
+
+    ***Challenge***
+
+    There are two similar methods, `StartsWith` and `EndsWith` that also search for sub-strings in a string. These find a sub-string at the beginning or the
+    end of the string. Try to modify the sample above to use `StartsWith` and
+    `EndsWith` instead of `Contains`.
+- title: Complete challenge
+  durationInMinutes: 1
+  content: |
+    Did you come up with something like this?
+
+    ```csharp
+    string message = "You say goodbye, and I say hello";
+    Console.WriteLine(message.StartsWith("You"));
+    Console.WriteLine(message.StartsWith("goodbye"));
+
+    Console.WriteLine(message.EndsWith("hello"));
+    Console.WriteLine(message.EndsWith("goodbye"));
+    ```
+- content: |
```

Comment:
another feedback about interactive: should Next steps, Next tutorial be sentence case?

Reply:
/cc @EisenbergEffect